src/components/entity/ha-statistic-picker.ts

```diff
@@ -122,6 +130,19 @@ export class HaStatisticPicker extends LitElement {
           includeUnitClasses.includes(meta.unit_class)
         );
       }
+      if (includeDeviceClass) {
+        const includeDeviceClasses: (string | null)[] =
+          ensureArray(includeDeviceClass);
+        statisticIds = statisticIds.filter((meta) => {
+          const stateObj = this.hass.states[meta.statistic_id];
+          if (!stateObj) {
+            return true;
+          }
```

Comment:
Why would we do this ? Seems like if a statistic is no longer in the state machine and so can't be checked, we don't have to show it? 

Not sure how common, we are restoring anything with a unique ID, and so state always available.

Reply:
We also have external statistics